### ISSUE TITLE
Consider localizations for both client-side and server-side

### ISSUE THREAD

**Issue by CathairDoherty** [RhubarbPHP/Rhubarb#5]
As some open source frameworks are used a crossed the world, we will need to look at localizations on both the client side and server side so that sites are.  We will also need to look at WCAG and UAAG and ATAG guidelines

e.g.
 on the client side uses i18n javascript to translate words or phases
 using moment.js from dates

**Comment by acuthbert** [comment]
Yes, a sound idea. We would just need to have a way to say what languages we want to deploy so that the translation files don't take too long to load if most of them aren't needed for the application.
